frontend/www/js/omegaup/arena/arena.js

```diff
@@ -256,11 +270,65 @@ export class Arena {
     // The Markdown-to-HTML converter.
     self.markdownConverter = UI.markdownConverter();
 
-    // Currently opened notifications.
-    self.notifications = new Notifications();
-    OmegaUp.on('ready', function() {
-      self.notifications.attach($('#notifications'));
-    });
+    // Currently opened clarification notifications.
+    self.commonNavbar = null;
+    if (document.getElementById('common-navbar')) {
+      self.commonNavbar = new Vue({
+        el: '#common-navbar',
+        render: function(createElement) {
+          return createElement('omegaup-common-navbar', {
+            props: {
+              header: this.header,
+              graderInfo: this.graderInfo,
+              graderQueueLength: this.graderQueueLength,
+              errorMessage: this.errorMessage,
+              initialClarifications: this.initialClarifications,
+            },
+          });
+        },
+        data: {
+          header: self.options.headerPayload,
+          graderInfo: null,
+          graderQueueLength: -1,
+          errorMessage: null,
+          initialClarifications: [],
+        },
+        components: {
+          'omegaup-common-navbar': common_Navbar,
+        },
+      });
+
+      if (self.options.headerPayload.isAdmin) {
+        API.Notification.myList({})
+          .then(function(data) {
```

Comment:
```suggestion
          .then(data => {
```

por consistencia con L310.

Reply:
Listo.